### PR TITLE
Update master

### PR DESCRIPTION
**Pull request by zFernand0**
- Fix Jenkinsfile references
- Point npmrc to giza artifactory
- update requred snapshots
- Moved one system test to integration
- Quickly implemented the skipProperties functionality :smile:
- Fixed all vulns
- Update coverage targets (lower so that it can pass)
- remove registry URL to install the `@zowe/cli` from public npm

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,2 +1,2 @@
 registry=http://registry.npmjs.org/
-@zowe:registry=https://api.bintray.com/npm/ca/brightside/
+@zowe:registry=https://gizaartifactory.jfrog.io/gizaartifactory/api/npm/npm-release/
```
Comment (tucker01) [inline .npmrc]: Does this have to be giza now that its on public npm? Won't this fail for most users?

Also, I've seen in the APIML repo that they include an access key here... this way folks can at least download from giza.

File: __tests__/__src__/environment/TestEnvironment.ts [hunk 2]
```diff
@@ -52,7 +52,10 @@ export class TestEnvironment {
         // Get a unique test data area
         const testDirectory: string = TestEnvironment.createUniqueTestDataDir(params.testName);
 
-        const systemProps = TestEnvironment.loadSystemTestProperties(undefined, testDirectory);
+        let systemProps;
+        if (!params.skipProperties) {
+            TestEnvironment.loadSystemTestProperties(undefined, testDirectory);
```
Comment (dogboydog) [inline __tests__/__src__/environment/TestEnvironment.ts]: Looks good

File: .npmrc (hunk 1 above)
Comment (zFernand0) [reply .npmrc]: I agree with pointing to public NPM (i.e. removing the zowe scope declaration all-together)

And I was under the impression that that was sort of illegal/wrong to open giza to public access, hence why we never included one.

I will remove the zowe scope form there and re-run tests.
Thanks.

File: __tests__/__src__/environment/TestEnvironment.ts (hunk 2 above)
Comment (zFernand0) [reply __tests__/__src__/environment/TestEnvironment.ts]: I messed up here since I didn't even assign the systemProps variable.
Whoops. 
Will fix it now

Comment (dogboydog) [reply __tests__/__src__/environment/TestEnvironment.ts]: Haha whoops

File: .npmrc (hunk 1 above)
Comment (tucker01) [reply .npmrc]: That is great thanks. I'm not sure if its cool to do this or not, but here is the sample:
https://github.com/zowe/sample-spring-boot-api-service/blob/master/gradle.properties

**Comment by tucker01** [comment]
Can I merge this @zFernand0 ?

**Comment by zFernand0** [comment]
> Can I merge this @zFernand0 ?

yup

**Pull request merged**